test/10-verify.spec.js

```diff
@@ -808,6 +808,40 @@ for(const [version, mockCredential] of versionedCredentials) {
           .contain('"issuer" must be a URI');
       });
 
+      it('should reject credentialStatus that is not a URI', () => {
+        const credential = mockCredential();
+        credential.credentialStatus = {
+          id: 'not-a-url',
+          type: 'urn:type'
+        };
+        let error;
+        try {
+          vc._checkCredential({credential});
+        } catch(e) {
+          error = e;
+        }
+        should.exist(error,
+          'Should throw error when "evidence" is not a URI');
```

Comment:
What's `"evidence"` here? Looks like a copy/paste error.

Reply:
I actually was assuming as `evidence` is moving out the spec that `credentialStatus` was now a piece of evidence? @davidlehn is this a mistake or should the `credentialStatus.id` not being a URI be taken as an evidence error?

Reply:
Looks like typo.  Fixed.  Do these id changes need a changelog entry?  Not sure what's going in this for this PR.